src/main/scala/org/ergoplatform/settings/VotingTargets.scala

```diff
@@ -0,0 +1,37 @@
+package org.ergoplatform.settings
+
+import com.typesafe.config.Config
+import net.ceedubs.ficus.Ficus._
+import org.ergoplatform.settings.ErgoSettings.configPath
+
+import scala.collection.JavaConverters._
+
+/**
+  * Local miner settings to determine, how to vote
+  *
+  * @param targets       - desired parameters targets
+  * @param desiredUpdate - rules to deactivate if sof-fork is desirable
+  */
+case class VotingTargets(targets: Map[Byte, Int], desiredUpdate: ErgoValidationSettingsUpdate) {
+  val softFork: Int = targets.getOrElse(Parameters.SoftFork, 0)
+}
+
+object VotingTargets {
+
+  val empty: VotingTargets = VotingTargets(Map(), ErgoValidationSettingsUpdate.empty)
+
+  def fromConfig(config: Config): VotingTargets = {
+    val toDisable = config.as[Array[Int]](s"$configPath.voting.${Parameters.SoftForkDisablingRules}")
+    val votingObject = config.getObject(s"$configPath.voting")
+
+    val parameterTargets = votingObject
+      .keySet()
+      .asScala
+      .filter(_.toByte != Parameters.SoftForkDisablingRules)
+      .map(id => id.toByte -> votingObject.get(id).render().toInt)
+      .toMap
+    val desiredUpdate = ErgoValidationSettingsUpdate(toDisable.map(_.toShort), Seq())
```

Comment:
I expect, that on proposing a soft-fork developers will release a version with the hard-coded `desiredUpdate`, and if anyone is going to vote for the soft-fork it will install this software version.
I may do it's configuration now, but it is a local thing so may be defined at any time in future.